eventloop/eventloop.go

```diff
@@ -49,6 +58,18 @@ func NewEventLoop() *EventLoop {
 	return loop
 }
 
+type option func(*EventLoop)
```

Comment:
Any reason for this being unexported? I suspect linters would complain. Also this would prevent custom external options which could be useful...

Reply:
Well EventLoop has no exported fields so there isn't much (currently) a custom external option could do to change the behavior of NewEventLoop, but otherwise I don't see why this couldn't be exported.  I'm happy to update it if you feel that'd be more forward-looking.

Reply:
Yes please

Reply:
Done!